android/src/main/java/ie/gov/tracing/nearby/riskcalculation/RiskCalculationV2.java

```diff
@@ -242,7 +214,9 @@ private ExposureEntity buildExposureEntity(List<DailySummary> dailySummaries, Li
         if (config.getContiguousMode()) {
             List<WindowData> exceeded = filterForExceededWindows(windowItems);
             if (exceeded.size() > 0) {
-                long dayVal = Instant.ofEpochMilli(exceeded.get(0).getDate()).atZone(ZoneId.systemDefault()).toLocalDate().toEpochDay();
+                Calendar cal1 = Calendar.getInstance();
```

Comment:
thanks @colmharte for the change

Not sure if we need the Calendar API here since local dates don't apply.
I think the ENS dailySummaries & daysSinceLastExposure are always grouped by UTC days. 
Would this work? `TimeUnit.DAYS.convert(exceeded.get(0).getDate(), TimeUnit. MILLISECONDS);`

Reply:
Yes have updated to remove calendar